plugins/GrapesJsBuilderBundle/Controller/GrapesJsController.php

```diff
@@ -283,17 +283,10 @@ function ($a, $b) {
 
     private function checkForMjmlTemplate($template)
```

Comment:
@escopecz this function is used in this [Controller](https://github.com/mautic/mautic/blob/5.x/plugins/GrapesJsBuilderBundle/Controller/GrapesJsController.php#L90) according to the code this function should not be used.

Reply:
It should be marked as deprecated then?

Reply:
I just marked it